pkgs/applications/misc/pipewire-screenaudio/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, gawk
+, hexdump
+, jq
+, pipewire
+, psmisc
+}:
+stdenv.mkDerivation rec {
+  pname = "pipewire-screenaudio";
+  version = "0.2.0";
+
+  src = fetchFromGitHub {
+    owner = "IceDBorn";
+    repo = "pipewire-screenaudio";
+    rev = version;
+    hash = "sha256-IfPW0qmIUMIuevMLolYyKpYMBiiBG1OJA7/Wtxp+EzM=";
+  };
+
+  buildInputs = [
+    gawk
+    hexdump
+    jq
+    pipewire
+    psmisc
+  ];
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/lib/out
+    install -Dm755 native/connector/pipewire-screen-audio-connector.sh $out/lib/connector/pipewire-screen-audio-connector.sh
+    install -Dm755 native/connector/virtmic.sh $out/lib/connector/virtmic.sh
+
+    # Firefox manifest
+    sed -i "s|/usr/lib/pipewire-screenaudio|$out/lib|g" native/native-messaging-hosts/firefox.json
+    install -Dm644 native/native-messaging-hosts/firefox.json $out/lib/mozilla/native-messaging-hosts/com.icedborn.pipewirescreenaudioconnector.json
+
+    runHook postInstall
+  '';
```

Comment:
Can you check all these inside `postInstall` instead?

Reply:
I think that it will work just fine but why is that change necessary? Is it just because it just installs files instead of running actual "building" steps?